### ISSUE TITLE
no-outlet-outside-routes error showing for route template

### ISSUE THREAD

**Issue by marshy2201** [ember-template-lint/ember-template-lint#700]
The no-outlet-outside-routes error is showing for a file that is a route template.

Is this because of the file directory where 'components' is named?

```hbs
{{!-- app/templates/products/components/dc.hbs --}}
{{title "DC" replace=false prepend=false}}
{{outlet}}
```

**Comment by marshy2201** [comment]
Here is the errors that are showing up:

![eslint-errors](https://user-images.githubusercontent.com/20504912/61528038-ba692980-aa15-11e9-83ba-c84285d049ae.png)

**Comment by rwjblue** [comment]
I believe this was fixed by #762
